samples/Azure.Management.Storage/Generated/FileServicePropertiesContainer.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why did this become FileServicePropertiesContainer vs FileServiceContainer

Reply:
The ResourceName we found in swagger is "FileServiceProperties" instead of "FileService": https://github.com/Azure/azure-rest-api-specs/blob/e606243e5297312781dd7dbfd7ab76d2329cc088/specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json#L94